agent/util-scripts/unittests

```diff
@@ -21,12 +21,16 @@ export _testdir=${_testroot}/pbench
 # Copy util-scripts execution environment to _testroot
 _testopt=${_testroot}/opt/pbench-agent
 res=0
-mkdir -p ${_testopt}/config ${_testopt}/util-scripts ${_testopt}/tool-scripts ${_testopt}/lib ${_testopt}/unittest-scripts
+mkdir -p ${_testopt}/config ${_testopt}/util-scripts ${_testopt}/tool-scripts ${_testopt}/lib ${_testopt}/unittest-scripts ${_testopt}/common/bin ${_testopt}/common/lib/configtools
 let res=res+${?}
 cp -L ${_tdir}/../base ${_testopt}/
 let res=res+${?}
 cp -rL ${_tdir}/../config/* ${_testopt}/config/
 let res=res+${?}
+cp -a $_tdir/../../bin/getconf.py $_testopt/common/bin
+let res=res+${?}
+cp -a $_tdir/../../lib/configtools/__init__.py $_testopt/common/lib/configtools
+let res=res+${?}
```

Comment:
All the comments in `bench-scripts/unittests` are applicable here too.

Reply:
ok